apps/public-docsite-v9/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -219,7 +256,7 @@ export const FluentDocsPage = () => {
             {videos && <VideoPreviews videos={videos} />}
           </div>
           <RenderPrimaryStory primaryStory={primaryStory} skipPrimaryStory={skipPrimaryStory} />
-          <RenderArgsTable primaryStory={primaryStory} hideArgsTable={hideArgsTable} />
+          <RenderArgsTable story={primaryStory} hideArgsTable={hideArgsTable} argTypes={primaryStoryContext.argTypes} />
```

Comment:
these argTypes are properly transformed via preview enhance in preview.js although we cannot use them it seems

**Task:**
- [ ] revert this argTypes prop change